library/src/main/java/com/google/android/exoplayer2/extractor/mp4/FragmentedMp4Extractor.java

```diff
@@ -1192,7 +1192,7 @@ private static DrmInitData getDrmInitDataFromAtoms(List<Atom.LeafAtom> leafChild
         if (uuid == null) {
           Log.w(TAG, "Skipped pssh atom (failed to extract uuid)");
         } else {
-          schemeDatas.add(new SchemeData(uuid, MimeTypes.VIDEO_MP4, psshData));
+          schemeDatas.add(new SchemeData(uuid, MimeTypes.CENC, psshData));
```

Comment:
This doesn't appear to be necessary? I have a hunch it might break Widevine/PlayReady on some earlier API levels also, although I didn't confirm this. If it works without, can we just leave it how it was and also remove the CENC constant from MimeTypes.java?